frontend/cypress/e2e/createANewProject.cy.ts

```diff
@@ -2,7 +2,31 @@ import homePage from '../pages/home'
 import configPage from '../pages/metrics/config'
 import metricsPage from '../pages/metrics/metrics'
 import reportPage from '../pages/metrics/report'
-
+const cycleTimeData = [
```

Comment:
Maybe we can hold a Zoom meeting to talk this part
you can touch me on slack anytime